tools/confix/data/v0.51-client.toml

```diff
@@ -0,0 +1,22 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                          ###
+###############################################################################
+
+# The network chain ID
+chain-id = "demo"
+# The keyring's backend, where the keys are stored (os|file|kwallet|pass|test|memory)
+keyring-backend = "test"
+# Default key name, if set, defines the default key to use for signing transaction when the --from flag is not specified
+keyring-default-keyname = ""
+# CLI output format (text|json)
+output = "text"
+# <host>:<port> to CometBFT RPC interface for this chain
+node = "tcp://localhost:26657"
+# Transaction broadcasting mode (sync|async)
+broadcast-mode = "sync"
+# This is default the gas adjustment factor used in tx commands.
+# It can be overwriten by the --gas-adjustment flag in each tx command.
+gas-adjustment = 1.5
```

Comment:
Let's remove this one. This is not an addition of v0.51, but a demonstration of client.toml in simapp: https://github.com/cosmos/cosmos-sdk/blob/912390d5fc4a32113ea1aacc98b77b2649aea4c2/simapp/simd/cmd/config.go#L29